creator-node/src/services/stateMachineManager/makeOnCompleteCallback.ts

```diff
@@ -75,7 +75,9 @@ function makeOnCompleteCallback(
     // Bull serializes the job result into redis, so we have to deserialize it into JSON
     let jobResult: AnyDecoratedJobReturnValue
     try {
-      logger.info(`Job successfully completed. Parsing result: ${resultString}`)
+      logger.debug(
```

Comment:
this was the gigantic log that printed stuff for all the bull queues

Reply:
makes sense to move to debug for now, i think we can just remove altogether tbh, since its already recorded in the bull dashboard
but our disk pressure isn't coming from logs so no concern there

Reply:
ah actually @theoilie already made this change to remove the massive resultString separately